subsys/emul/i2c/emul_sbs_gauge.c

```diff
@@ -61,15 +62,15 @@ static int reg_read(const struct emul *target, int reg)
 	case SBS_GAUGE_CMD_CYCLE_COUNT:
 	case SBS_GAUGE_CMD_DESIGN_VOLTAGE:
 		/* Arbitrary stub value. */
-		val = 1;
+		*val = 1;
 		break;
 	default:
 		LOG_ERR("Unknown register 0x%x read", reg);
 		return -EIO;
 	}
 	LOG_INF("read 0x%x = 0x%x", reg, val);
```

Comment:
```
/__w/zephyr/zephyr/subsys/emul/i2c/emul_sbs_gauge.c: In function 'emul_sbs_gauge_reg_read':
/__w/zephyr/zephyr/subsys/emul/i2c/emul_sbs_gauge.c:71:17: error: format '%x' expects argument of type 'unsigned int', but argument 3 has type 'int *' [-Werror=format=]
   71 |         LOG_INF("read 0x%x = 0x%x", reg, val);
      |                 ^~~~~~~~~~~~~~~~~~       ~~~
      |                                          |
      |                                          int *
```

Reply:
Fixed, but for some reason I didn't get this build failure locally. Do you have yours setup with `clang-tidy`?

Reply:
Yeah I saw it on the CI run, one step was failing.